press/api/site.py

```diff
@@ -1854,28 +1852,26 @@ def get_upload_link(file, parts=1):
 
 @frappe.whitelist()
 def multipart_exit(file, id, action, parts=None):
+	press_settings: "PressSettings" = frappe.get_single("Press Settings")  # type: ignore
+	bucket_name = "uploads.frappe.cloud"  # TODO: Use press_settings.remote_uploads_bucket
```

Comment:
:eyes:

Reply:
If the Frappe team can confirm that the current config on Frappe Cloud is indeed `remote_uploads_bucket =  "uploads.frappe.cloud"`

```suggestion
	bucket_name = press_settings.remote_uploads_bucket
```